src/Sulu/Bundle/DocumentManagerBundle/DependencyInjection/SuluDocumentManagerExtension.php

```diff
@@ -13,14 +13,56 @@
 
 use Symfony\Component\Config\FileLocator;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\DefinitionDecorator;
 use Symfony\Component\DependencyInjection\Extension\Extension;
 use Symfony\Component\DependencyInjection\Extension\PrependExtensionInterface;
 use Symfony\Component\DependencyInjection\Loader\XmlFileLoader;
+use Symfony\Component\DependencyInjection\Reference;
 
 class SuluDocumentManagerExtension extends Extension implements PrependExtensionInterface
 {
     public function prepend(ContainerBuilder $container)
     {
+        // process the configuration of SuluCoreExtension
+        $configs = $container->getExtensionConfig($this->getAlias());
+        $parameterBag = $container->getParameterBag();
+        $configs = $parameterBag->resolveValue($configs);
+        $config = $this->processConfiguration(new Configuration(), $configs);
+
+        foreach ($config['sessions'] as &$session) {
+            // Do not automatically "login" workspaces upon repository instantiation.
+            // This allows "admin" commands to be executed (f.e. for creating workspaces).
+            // NOTE: we may be able to remove this when we upgrade to DoctrinePhpcrBundle 1.3
+            if (isset($session['backend'])) {
+                $session['backend']['check_login_on_server'] = false;
+            }
+        }
+
+        foreach (array_keys($container->getExtensions()) as $name) {
```

Comment:
I assume because there is no `getExtension($name)` method. This logic was copied verbatim from the `SuluCoreBundle`, which was inturn copied from the `CmfCoreBundle`. Will check to see if it can now be improved.